lib/caiquenator_web/controllers/phrase_controller.ex

```diff
@@ -0,0 +1,13 @@
+defmodule CaiquenatorWeb.PhraseController do
+  use CaiquenatorWeb, :controller
+
+  @phrases File.cwd!()
```

Comment:
You can use `@external_resource` here (https://hexdocs.pm/elixir/Module.html#module-external_resource) this way when the file is changed you won't need to recompile the application.

Reply:
it was an @josevalim tip xD

Reply:
Done. Can you check if I did it right? 😆